examples/chip-tool-darwin/templates/commands.zapt

```diff
@@ -155,6 +159,7 @@ public:
         {{else}}
         AddArgument("attr-value", {{asTypeMinValue type}}, {{asTypeMaxValue type}}, &mValue);
         {{/if_chip_complex}}
+        AddArgument("timedWriteTimeoutMs", 0, UINT16_MAX, &mTimedWriteTimeoutMs);
```

Comment:
This is not API-compatible with `chip-tool`, which uses `timedInteractionTimeoutMs` for writes.  Is that purposeful, or an oversight?

Reply:
This was definitely an oversight. Thank you for the catch. I will update it to be timedInteractionTimeoutMs.

Reply:
Updated.